src/nepiada/env/nepiada.py

```diff
@@ -236,7 +236,7 @@ def reset(self, seed=None, options=None):
         Returns the observations for each agent
         """
         self.agents = self.possible_agents[:]
-        print("All Agents: ", str(self.agents))
+        print("NEPIADA INFO: All Agents: ", str(self.agents))
```

Comment:
We should look into using a proper debug print messages using modules like logging to easily toggle messages ON and OFF

Reply:
Yep thats true, I think that would be a future PR though. I think this is fine for now personally as these are just sanity prints rather than debugging.